Library/ViewModels/LiveStreamDiscoveryLiveNowCellViewModel.swift

```diff
@@ -48,6 +50,12 @@ LiveStreamDiscoveryLiveNowCellViewModelInputs, LiveStreamDiscoveryLiveNowCellVie
     self.creatorLabelText = liveStreamEvent
       .map { Strings.Creator_name_is_live_now(creator_name: $0.creator.name) }
 
+    self.numberPeopleWatchingHidden = liveStreamEvent.map { $0.numberPeopleWatching == nil }
```

Comment:
from slack convo we could add one additional condition for `$0.numberPeopleWatching == nil || $0.numberPeopleWatching == .some(0)` so that we don't show a said `0` label